package.json

```diff
@@ -5,7 +5,7 @@
   "name": "ledger-live-desktop",
   "productName": "Ledger Live",
   "description": "Ledger Live - Desktop",
-  "version": "2.30.0",
+  "version": "2.31.0",
```

Comment:
can you rewrite your PR so it doesn't include rewritten commit history? thanks.

Reply:
Sure thing, done ✅